ansible_collections/arista/avd/roles/eos_designs/doc/management-settings-v4.0.md

```diff
@@ -82,7 +82,7 @@ Gives ability to monitor and react to Syslog messages provides a powerful and fl
 
 ```yaml
 event_handlers:
-  < event_handler_name >:
+  - name: < event_handler_name >
```

Comment:
This change is technically not related to the PR (local-users) and there is no code in the PR modifying the behavior - is it expected?

Reply:
its not in code, but I found it with a variable key so updated it.

Reply:
its by mistake, I am creating a separate PR for event_handlers.